contracts/libs/StringUtil.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.5.16;
+
+
+library StringUtil {
+
+    function hashCompareWithLengthCheck(string memory _a, string memory _b) internal returns (bool) {
+
+        if (bytes(_a).length != bytes(_b).length) {
+            return false;
+        } else {
+            return keccak256(abi.encodePacked(_a)) == keccak256(abi.encodePacked(_b));
```

Comment:
I'm proabably missing a point, but is this not equivalent to `abi.encodePacked(_a) == abi.encodePacked(_b)`?

Reply:
the compiler does not like it :
`TypeError: Operator == not compatible with types bytes memory and bytes memory`